### PR TITLE
Editorial: fix cross refs, tidy up

### PR DESCRIPTION
**Pull request by marcoscaceres**
Editors, reviewers, and wg members, I would kindly ask that you perhaps put in place a some means to double check your references before merging pull requests. This can be as simple as just using a Github [pull request template](https://help.github.com/articles/creating-a-pull-request-template-for-your-repository/) with a checkbox that someone has done this. 

Also, please try to be precise with what you reference. For instance, this spec tried to reference "dedicated workers" and "shared workers", which are not actual concepts in HTML (those are non-normative concepts used in examples and tutorials). Those concepts are actually represented by IDL interfaces, which is what you actually want to reference. 

Please also try to make more extensive use of HTML5 tidy before merging:

```
tidy -config tidyconfig.txt -o index.html index.html
```

That will keep your document more maintainable, and it will make it easier to compare and track changes.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/hr-time/pull/61.html" title="Last updated on Oct 31, 2018, 7:50 AM GMT (0379cdf)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/hr-time/61/0f77f49...0379cdf.html" title="Last updated on Oct 31, 2018, 7:50 AM GMT (0379cdf)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -140,12 +130,12 @@ <h2>Introduction</h2>
...
-      origin</a>'s. To display such events on the same timeline, the
-      application can translate the <a>DOMHighResTimeStamp</a>'s with the help
-      of the <a>performance.timeOrigin</a> attribute.</p>
-      <pre class="example highlight">
+      application, including events from <a>Worker</a> or <a>SharedWorker</a>,
+      which have different <a>time origin</a>'s. To display such events on the
+      same timeline, the application can translate the
+      <a>DOMHighResTimeStamp</a>'s with the help of the
+      <a>performance.timeOrigin</a> attribute.</p>
+      <pre class="example js">
```
Comment (marcoscaceres) [inline index.html]: Note: you can indent `pre` stuff... no need to put it flush to the left.

File: index.html [hunk 2]
```diff
@@ -394,36 +399,28 @@ <h3>Clock drift</h3>
...
+    Karen Anderson, Nat Duca, Philippe Le Hegaret, Ryosuke Niwa, Simha
+    Sethumadhavan, Todd Reifsteck, Tony Gentilcore, Vasileios P. Kemerlis, Yoav
+    Weiss, and Yossef Oren for their contributions to this work.</p>
   </section>
-
 </body>
 </html>
-
-<!-- spec references. preserve before running tidy! -->
-[DOMTimeStamp]: https://heycam.github.io/webidl/#DOMTimeStamp
```
Comment (marcoscaceres) [inline index.html]: I don't suggest doing this... use `data-cite=` on `dfn` and `a` elements instead.

File: index.html [hunk 3]
```diff
@@ -199,49 +189,51 @@ <h3>Examples</h3>
...
+          <li>otherwise, the time of starting the <a data-cite="HTML#navigate">
+            navigation</a> responsible for loading the <a data-cite=
+            "HTML#concept-document-window"><code>Window</code> object's newest
+            Document object</a>.
           </li>
         </ul>
       </li>
-      <li>If the [global object] is a <code>[WorkerGlobalScope]</code> object, the <a>time origin</a> MUST
-        be equal to the [official moment of creation] of the worker.
+      <li>If the <a>global object</a> is a <code>[WorkerGlobalScope]</code>
```
Comment (yoavweiss) [inline index.html]: do we need the brackets here?

File: index.html [hunk 4]
```diff
@@ -199,49 +189,51 @@ <h3>Examples</h3>
...
+    timestamp</a> given a <a>global object</a> (<var>global</var>):</p>
     <ul>
-      <li>Assert that <var>global</var>'s <a>time origin</a> is not
-      undefined.</li>
+      <li>Assert that <var>global</var>'s <a>time origin</a> is not undefined.
+      </li>
       <li>Let <var>t1</var> be the <a>DOMHighResTimeStamp</a> representing the
-      high resolution [Unix time] at which the <a>global monotonic clock</a> is
-      zero.</li>
+      high resolution <a href="https://en.wikipedia.org/wiki/Unix_time">Unix
```
Comment (yoavweiss) [inline index.html]: There's no better place to ref than wikipedia here?

File: index.html [hunk 5]
```diff
@@ -340,17 +339,20 @@ <h3>Clock resolution</h3>
...
-      [[CACHE-ATTACKS]].</p>
-      <p data-tests='timing-attack.html'>This specification defines an API that provides sub-millisecond time
-      resolution, which is more accurate than the previously available
-      millisecond resolution exposed by [DOMTimeStamp]. However, even without
-      this new API an attacker may be able to obtain high-resolution estimates
-      through repeat execution and statistical analysis. To ensure that the new
-      API does not significantly improve the accuracy or speed of such attacks,
-      the recommended minimum resolution of the <a>DOMHighResTimeStamp</a> type
-      should be inaccurate enough to prevent attacks.</p>
+      [[?CACHE-ATTACKS]].</p>
```
Comment (yoavweiss) [inline index.html]: Why is the '?' needed here?

Comment (marcoscaceres) [reply index.html]: My reading was that CACHE-ATTACKS is cited in a non-normative way. The "?" makes it non-normative in the bibliography.

File: index.html [hunk 6]
```diff
@@ -394,36 +399,28 @@ <h3>Clock drift</h3>
       available entropy about the user.</p>
     </section>
   </section>
+  <section>
+    <h2>Dependencies</h2>
+    <p>The [[HTML]] specification defines the following the following:
```
Comment (yoavweiss) [inline index.html]: "the following" is here twice

File: index.html (hunk 4 above)
Comment (marcoscaceres) [reply index.html]: It could definitely be added to the localBiblio.

File: index.html (hunk 3 above)
Comment (marcoscaceres) [reply index.html]: Oh woopsy. Missed that one. Fixing.

File: index.html (hunk 4 above)
Comment (marcoscaceres) [reply index.html]: ah, sorry, I misunderstood. I'm not sure what might be better, tbh.... maybe ECMAScript's Date object points to a better reference?

File: index.html (hunk 3 above)
Comment (yoavweiss) [reply index.html]: Thank you! :)

File: index.html (hunk 6 above)
Comment (marcoscaceres) [reply index.html]: There is something poetic about "the following the following"... alas... will fix. Le sigh 👨‍🎨.

File: index.html (hunk 4 above)
Comment (yoavweiss) [reply index.html]: Let me rephrase: Is there an IETF/W3C/WHATWG/Other standard definition of unix time? Relying on wikipedia seems a bit fragile

Comment (marcoscaceres) [reply index.html]: So, you probably want one of the POSIX standards, which are actual IEEE standards (not to cite wikipedia again: https://en.wikipedia.org/wiki/POSIX) - but one of those will define "unix time". However, I've never looked at any of those.

Comment (yoavweiss) [reply index.html]: Seems like we could buy ISO8601 for measly 138CHF to find out if it's better defined there...

Comment (yoavweiss) [reply index.html]: ECMAScript define their own: https://www.ecma-international.org/ecma-262/9.0/index.html#sec-date-objects

Comment (marcoscaceres) [reply index.html]: Will leave that to you all to figure out :)

Comment (yoavweiss) [reply index.html]: I *think* we can replace "Unix-Time" with "Time" and point at the ES definition. Thoughts?

Comment (yoavweiss) [reply index.html]: Maybe it'd be best to land this and iterate on the "unix-time" issue

Comment (yoavweiss) [reply index.html]: Missed the fact that we were already citing wikipedia. That makes "landing this and iterate later" a no-brainer

File: index.html (hunk 5 above)
Comment (yoavweiss) [reply index.html]: OK, cool
